### DISCUSSION TITLE
alist无法进入

### DISCUSSION THREAD

**Discussion by LYL-0** [AlistGo/alist#6710] [General]
D:\app\alist\alist-windows_3.33.0_amd64>alist server
INFO[2024-07-06 16:10:04] reading config file: data\config.json
FATA[2024-07-06 16:10:04] update config struct error: open data\config.json: Access is denied.
有人可以帮我解答一下吗 浏览器就不去alist ,运行的话显示如上

**Comment by LYL-0** [comment]
![微信图片_20240706163729](https://github.com/alist-org/alist/assets/174760387/5540d4f3-cde4-49f3-8839-2054296329dd)

**Comment by fredericky123** [comment]
我也遇到了这个问题 请问如何解决呢

**Comment by Kikyo66** [comment]
啊啊啊啊我也是这样，头天部署完是正常使用的，第二天再打开就这样了，网上找了很多方法试了都没用，localhost可以正常打开，但是加上端口号就不行了

**Reply by Kikyo66** [reply]
啊啊啊我又试了一下，在另外一个磁盘新建了一个文件夹，把alist.exe丢进去重新运行又可以了！

**Reply by fredericky123** [reply]
所以必须要换文件夹了吗  不知道原因是啥
